### PR TITLE
Improve check for packages to update from #356

### PR DESCRIPTION
**Pull request by r0x0d**
This PR introduces some changes and additions to the PR#356 that were
observed after we rebased the PR#398 and the CI started failing.

Jira reference (If any): https://issues.redhat.com/browse/RHELC-595
Bugzilla reference (If any):

Signed-off-by: Rodolfo Olivieri <rolivier@redhat.com>

### CODE REVIEW

File: convert2rhel/checks.py
Comment (bocekm) [inline convert2rhel/checks.py:485]:
```suggestion
            "Not updating the packages may cause the conversion to fail."
            % (len(packages_to_update), " ".join(packages_to_update))
```
I'd expect linux admins to know ho to update packages.
Also I removed the "," between the listed packages so they can copy and paste it to the yum update command.

**Review by bocekm — CHANGES_REQUESTED** [review]

File: convert2rhel/checks.py
Comment (r0x0d) [reply convert2rhel/checks.py]: done.

File: convert2rhel/checks.py
Comment (bocekm) [inline convert2rhel/checks.py:474]:
```suggestion
            "There was an error while checking whether the installed packages are up-to-date. Having updated system is an important prerequisite for a successful conversion. Consider stopping the conversion to verify that manually."
```

**Review by bocekm — CHANGES_REQUESTED** [review]

File: convert2rhel/checks.py
Comment (r0x0d) [reply convert2rhel/checks.py]: done.

**Comment by bocekm** [comment]
> This pull request **introduces 2 alerts** when merging [6d4171b](https://github.com/oamg/convert2rhel/commit/6d4171bfaf6cffa0464fce8cc643559e95bd5b8f) into [25979b3](https://github.com/oamg/convert2rhel/commit/25979b362799d6b30a8df18e6a196e1f709d88a0) - [view on LGTM.com](https://lgtm.com/projects/g/oamg/convert2rhel/rev/pr-e3bbc0a9e954ebbd6bdd396f601dac6122e8a0f4)
> 
> **new alerts:**
> 
>     * 2 for Unused import

@r0x0d has added `# lgtm[py/unused-import]` but LGTM.com says that _"theses changes will only be effective once that pull request is merged."_

**Review by bocekm — APPROVED** [review]

**Review by bocekm — APPROVED** [review]

**Comment by danmyway** [comment]
All the checks have passed now, @r0x0d, @bocekm 
There was a task created to optimize the integration tests not passing `-y` option.

**Review by bocekm — APPROVED** [review]

**Pull request merged**
